wasm/src/language_service.rs

```diff
@@ -58,7 +60,35 @@ impl LanguageService {
                 )
                 .expect("callback should succeed");
         };
-        let mut worker = self.0.create_update_worker(diagnostics_callback, host);
+
+        let test_callables_callback = test_callables_callback
```

Comment:
Is the initial call done on line 40 for the diagnostics callback unnecessary then? (Mine thinks it might be)